src/actions/MenuActions.js

```diff
@@ -3,6 +3,9 @@ import axios from 'axios';
 
 // REQUESTS
 
+let breadcrumbsCache = {};
```

Comment:
we don't need to invalidate it. when you refresh the page, it gets cleared. when you 're navigating, it's not changed.